jetty/src/main/scala/org/http4s/server/jetty/JettyBuilder.scala

```diff
@@ -173,6 +175,21 @@ sealed class JettyBuilder[F[_]] private (
     }
   }
 
+  private def updateClientAuth(
+      sslContextFactory: SslContextFactory,
+      clientAuthMode: SSLClientAuthMode.Value): Unit =
+    clientAuthMode match {
+      case SSLClientAuthMode.NotRequested =>
+        sslContextFactory.setWantClientAuth(false)
+        sslContextFactory.setNeedClientAuth(false)
+
+      case SSLClientAuthMode.Requested =>
+        sslContextFactory.setWantClientAuth(false)
```

Comment:
`true`, right?

I'm inferring from [the javadoc](https://docs.oracle.com/javase/8/docs/api/javax/net/ssl/SSLEngine.html#setNeedClientAuth-boolean-) that we don't need to set both booleans.

Reply:
Sorry. Will fix both.